db/migrate/20200417174053_add_unique_index_to_business_objectives.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueIndexToBusinessObjectives < ActiveRecord::Migration[5.2]
+  def change
+    add_index :business_objectives, :title, unique: true
```

Comment:
I don't think this is right, multiple accounts should be able to have multiple business objectives with the same title. I tried this and the moment you try to create a BO from an account that may not even have any yet, you get `:title=>["has already been taken"]`.

Reply:
Makes sense :+1: I can't think of a good way to validate title is unique within a profile via the DB, so I guess we'll skip this one.